sdk/tests/android/testapp/src/main/java/software/amazon/awssdk/iottest/MainActivity.kt

```diff
@@ -0,0 +1,6 @@
+package software.amazon.awssdk.iottest
+
+import androidx.appcompat.app.AppCompatActivity
+
+class MainActivity : AppCompatActivity() {
```

Comment:
What's the purpose of this file? If it's a stab for the future work, then we can keep it

Reply:
It gets built and installed with everything included in its gradle.build file (the samples src files, the sdk, the crt library) and then the instrumented tests get run against it. We upload both the app package built from this as well as the app package built from the tests and run them in parallel.